src/utils/keyify.ts

```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line max-len
+// taken from here: https://stackoverflow.com/questions/47062922/how-to-get-all-keys-with-values-from-nested-objects/47063174
+// Used to get all key paths of object
+const keyify = (obj: any, prefix = ''): string[] =>
```

Comment:
`const keyify = <Obj extends Record<string,Obj>>(obj: Obj, prefix = ''): string[] =>` 
might get rid of `any`.

Reply:
hmm doesnt seem to work

Reply:
to me that seems like an impossible type :D

Reply:
Added better typing